arbnode/batch_poster.go

```diff
@@ -603,8 +582,6 @@ func (b *BatchPoster) checkEspressoValidation(
 				log.Warn("skipped espresso verification due to hotshot failure", "pos", b.building.msgCount)
 				return nil
 			}
-			// TODO: if current position is greater than the `skip`, should set the
```

Comment:
@ImJeremyHe  I am not sure why we had this so I deleted it but if it is important let me know